py3status/modules/mpd_status.py

```diff
@@ -213,7 +213,8 @@ def attr_getter(attr):
             text = "Failed to connect to mpd!"
         except CommandError:
             text = "Failed to authenticate to mpd!"
-            c.disconnect()
+
+        c.disconnect()
```

Comment:
maybe it would be best in a `else:` AND  in the `except CommandError` ? because if the connection fails the disconnect() will raise another Exception right ?